pkg/invoke/workflow.go

```diff
@@ -159,6 +160,10 @@ func (i *Invoker) rerunThread(ctx context.Context, c kclient.WithWatch, wf *v1.W
 
 	if thread.Status.CurrentRunName != "" || thread.Status.LastRunName != "" {
 		thread.Status.CurrentRunName = ""
+		if err := retention.SetLastUsedTime(ctx, c, &thread); err != nil {
+			return nil, nil, err
+		}
+
```

Comment:
nit: breaking up the assignments to empty strings here seems rubs me the wrong way.

Reply:
Fixed